### PR TITLE
Update storage.md

### PR DESCRIPTION
**Pull request by YvonneKallberg**
Issue #259  

This PR includes a new section 'How store your data after a project ends?' as well as suggestions that address the comments I've made in the issue about clarifying the question 'How to store your data during the course of your project?' I have also updated links to Data Sensitivity page.

### CODE REVIEW

**Comment by YvonneKallberg** [comment]
I now realize that I've updated security & privacy links to a page, Data sensitivity, that is not complete and thus not available on the web. Should the links to go Your domain - human data instead / in the meantime? The only thing missing is that it is not only human data that can be considered sensitive (but in reality I think most of the time it is human data that is the worry / main research focus).

**Comment by pinarpink** [comment]
Hi @YvonneKallberg  we will keep the `data_sensitivity` page in fact we decided to rename it to `data_classification`  according to the new structure and naming. In the data_classification page we will be able to talk about other kinds of sensitive data on that page. 

@bedroesb what is your suggestion, I think we can merge this PR before the overhaul and fix any broken links later on.

**Review by pinarpink — APPROVED** [review]

**Comment by bedroesb** [comment]
@YvonneKallberg and @pinarpink , yes, GDPR became Data protection and Data sensitivity became Data classification. I fixed this PR accordingly, can you check if I did it correct?

File: pages/your_problem/storage.md [hunk 1]
```diff
@@ -33,18 +33,17 @@ The timeline and purpose of the storage are also deciding factors for choosing a
 To find out what storage solutions are available to you and which one fits your needs, you can reach out to the IT-department of your host institution. Also, there are often national infrastructures available, for your domain or specialised on computing services, that offers storage solutions as well.
 
 For active data, it is important that it is easily accessible for everyone that needs to work with it. See recommendations on how to store and manage data during the course of your project below. For recommendations about storage after the active phase of a project, see recommendations further down.
-For sensitive data, consider the [Data Sensitivity guidelines](data_sensitivity).
+For sensitive data, consider the [Data Sensitivity guidelines](data_classification).
```
Comment (YvonneKallberg) [inline pages/your_problem/storage.md:36]: Change to [Data **Classification** guidelines] instead of [Data **Sensitivity** guidelines]

**Review by floradanna — CHANGES_REQUESTED** [review]
Thanks for this contribution!
Could you please add e!DAL and RDMP into the [tools and resources table](https://rdm.elixir-europe.org/tool_resource_update.html#making-changes)?

**Comment by bedroesb** [comment]
The respective tag you want to use for the tools in this case is `storage`

**Comment by YvonneKallberg** [comment]
I've added e!DAL and RDMP to the tools list.

**Review by floradanna — APPROVED** [review]

**Pull request merged**